### PR TITLE
jormun : add condition checking dead kraken

### PR DESCRIPTION
**Pull request by Hoshiyo**

### CODE REVIEW

File: fabfile/component/jormungandr.py [hunk 1]
```diff
@@ -239,6 +239,8 @@ def check_kraken_jormun_after_deploy(show=False):
             warn_list.append(dict(status='dead', region_id=item['region_id'], kraken_version=None))
         elif item['kraken_version'] != installed_kraken_version:
             warn_list.append(dict(status=item['status'], region_id=item['region_id'], kraken_version=item['kraken_version']))
+        elif item['status'] == "no_data":
+            warn_list.append(dict(status='no_data', region_id=item['region_id'], kraken_version=installed_kraken_version))
```
Comment (antoine-de) [inline fabfile/component/jormungandr.py]: not very important, but you can also do this with dict comprehension: 
```warn_list.append({status:'no_data', region_id:item['region_id'], kraken_version:installed_kraken_version})```

Comment (francois-vincent) [inline fabfile/component/jormungandr.py]: you meant dict literal, that should write:
warn_list.append({'status':'no_data', 'region_id':item['region_id'], 'kraken_version':installed_kraken_version}),
but writing it with a constructor is ok too :)

Comment (antoine-de) [inline fabfile/component/jormungandr.py]: yes sure it is just that it's quicker to write, but as stated not important (and no I meant dict comprehension :wink:  https://www.python.org/dev/peps/pep-0274/)

**Comment by antoine-de** [comment]
Can i merge this or do I need to wait for jenkins to tag it correctly ?

File: fabfile/component/jormungandr.py (hunk 1 above)
Comment (francois-vincent) [inline fabfile/component/jormungandr.py]: a dict literal is smth like {'a':1,'b':2,...} which is near-equivalent to a dict constructor, smth like dict(a=1, b=2, ...)
see: http://stackoverflow.com/questions/6610606/dict-literal-vs-dict-constructor-any-preferred
a dict comprehension is smth like {i : chr(65+i) for i in range(4)}, see pep 274
a dict comprehension feature an iterator
what we have here is a dict constructor/literal case, not a dict comprehension (no iteration)

**Comment by Hoshiyo** [comment]
not working yet

**Comment by antoine-de** [comment]
If it is not working, it is better to tag the PR as 'do_not_merge', not
avoid problems

File: fabfile/utils.py [hunk 2]
```diff
@@ -353,3 +353,22 @@ def start_or_stop_with_delay(service, delay, wait, start=True, only_once=False,
...
+
+    if warn_dict['jormungandr']:
+            status += "\nJormungandr version={}".format(warn_dict['jormungandr'])
+
+    version_sort = sorted(warn_dict['kraken'], key=lambda i: i.get('kraken_version'), reverse=True)
+    for item in version_sort:
+        status += "\nKraken {region_id} status={status} version={kraken_version}".format(**item)
+    if status:
+        status = "\n\n---------- Status" + status
+    print(yellow("{}".format(status)))
```
Comment (francois-vincent) [inline fabfile/utils.py]: print(yellow(status)) is equivalent and more simple

**Comment by TeXitoi** [comment]
@Hoshiyo when @antoine-de says "tags", he thinks "add a label".

**Pull request merged**